config/manifests/bases/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -104,5 +104,7 @@ spec:
   minKubeVersion: 1.27.0
   provider:
     name: Red Hat
-  replaces: cert-manager-operator.v1.15.1
-  version: 1.16.0
+  replaces: cert-manager-operator.v1.16.0
+  skips:
+  - cert-manager-operator.v1.15.1
+  version: 1.16.1
```

Comment:
Due to https://github.com/openshift/cert-manager-operator/pull/293#discussion_r2174286476